src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -323,6 +351,7 @@ public void HandleReadDataRateTimeout()
 
         public void OnInputOrOutputCompleted()
         {
+            // TODO poison streams?
```

Comment:
Can a Quic connection "Complete" before its streams do?

Reply:
I don't honestly know; that will be a question for the runtime team to figure out with their QUIC implementation 😄